configs/kubernetes-rules-configmap.yaml

```diff
@@ -0,0 +1,467 @@
+        - repository: apiserver
+          branch: master
+      - source:
+          repository: kubernetes
+          branch: release-1.7
+          dir: staging/src/k8s.io/apiextensions-apiserver
+        name: release-1.7
+        dependencies:
+        - repository: apimachinery
+          branch: release-1.7
+        - repository: client-go
+          branch: release-4.0
+        - repository: apiserver
+          branch: release-1.7
+      - source:
+          repository: kubernetes
+          branch: release-1.8
+          dir: staging/src/k8s.io/apiextensions-apiserver
+        name: release-1.8
+        dependencies:
+        - repository: apimachinery
+          branch: release-1.8
+        - repository: api
+          branch: release-1.8
+        - repository: client-go
+          branch: release-5.0
+        - repository: apiserver
+          branch: release-1.8
+      - source:
+          repository: kubernetes
+          branch: release-1.9
+          dir: staging/src/k8s.io/apiextensions-apiserver
+        name: release-1.9
+        dependencies:
+        - repository: apimachinery
+          branch: release-1.9
+        - repository: api
+          branch: release-1.9
+        - repository: client-go
+          branch: release-6.0
+        - repository: apiserver
+          branch: release-1.9
+      publish-script: /publish_scripts/publish_apiextensions_apiserver.sh
+    - destination: metrics
+      branches:
+      - source:
+          repository: kubernetes
+          branch: master
+          dir: staging/src/k8s.io/metrics
+        name: master
+        dependencies:
+        - repository: apimachinery
+          branch: master
+        - repository: api
+          branch: master
+        - repository: client-go
+          branch: master
+      - source:
+          repository: kubernetes
+          branch: release-1.7
+          dir: staging/src/k8s.io/metrics
+        name: release-1.7
+        dependencies:
+        - repository: apimachinery
+          branch: release-1.7
+        - repository: client-go
+          branch: release-4.0
+      - source:
+          repository: kubernetes
+          branch: release-1.8
+          dir: staging/src/k8s.io/metrics
+        name: release-1.8
+        dependencies:
+        - repository: apimachinery
+          branch: release-1.8
+        - repository: api
+          branch: release-1.8
+        - repository: client-go
+          branch: release-5.0
+      - source:
+          repository: kubernetes
+          branch: release-1.9
+          dir: staging/src/k8s.io/metrics
+        name: release-1.9
+        dependencies:
+        - repository: apimachinery
+          branch: release-1.9
+        - repository: api
+          branch: release-1.9
+        - repository: client-go
+          branch: release-6.0
+      publish-script: /publish_scripts/publish_metrics.sh
+    - destination: code-generator
+      branches:
+      - source:
+          repository: kubernetes
```

Comment:
if we have one bot per source repo, we can drop the repository of the source.

Reply:
it seems like we don't even use this value anywhere